docs/release_notes/v1.0.0-rc.1.md

```diff
@@ -0,0 +1,399 @@
+Runtime version: 1.0.0-rc.1
+```
+
+### Kubernetes
+
+#### Upgrading from previous version
+
+If you previously installed Dapr using Helm, you can upgrade Dapr to a new version.
+If you installed Dapr using the CLI, go [here](#starting-fresh-install-on-a-cluster).
+
+##### 1. Get the latest CLI
+
+Download the latest RC release from [here](https://github.com/dapr/cli/releases) and put the `dapr` binary in your PATH.
+
+##### 2. Upgrade existing cluster
+
+In this rc1 release, there is a new HA version of the placement service, that you need upgrade to. Upgrading from 0.11.x to 1.0.0-rc.1 safely requires an additional step to retain the 0.11.x placement service whilst migrating from the 0.11.x placement service to 1.0.0-rc.1 placement service. 
+
+First run these two commands to disable `helm upgrade` from uninstall 0.11.x placement service.
+
+```sh
+kubectl annotate deployment dapr-placement "helm.sh/resource-policy"=keep -n dapr-system
+kubectl annotate svc dapr-placement "helm.sh/resource-policy"=keep -n dapr-system
+```
+
+Then export certs manually.
+
+```sh
+dapr mtls export -o ./certs
+```
+
+Now, upgrade Dapr to 1.0.0-rc.1 using the below commands:
+
+```sh
+helm repo update
+helm upgrade dapr dapr/dapr --version 1.0.0-rc.1 --namespace dapr-system --reset-values --set-file dapr_sentry.tls.root.certPEM=./certs/ca.crt --set-file dapr_sentry.tls.issuer.certPEM=./certs/issuer.crt --set-file dapr_sentry.tls.issuer.keyPEM=./certs/issuer.key --set global.ha.enabled=true --wait
+```
+
+Once Dapr is installed completely, ensure that 0.11.x dapr-placement service is still running and **wait until all PODs are running**
+```sh
+kubectl get pods -n dapr-system -w
+
+NAME                                     READY   STATUS    RESTARTS   AGE
+dapr-dashboard-69f5c5c867-mqhg4          1/1     Running   0          42s
+dapr-operator-5cdd6b7f9c-9sl7g           1/1     Running   0          41s
+dapr-operator-5cdd6b7f9c-jkzjs           1/1     Running   0          29s
+dapr-operator-5cdd6b7f9c-qzp8n           1/1     Running   0          34s
+dapr-placement-5dcb574777-nlq4t          1/1     Running   0          76s  ---- 0.11.x placement
+dapr-placement-server-0                  1/1     Running   0          41s
+dapr-placement-server-1                  1/1     Running   0          41s
+dapr-placement-server-2                  1/1     Running   0          41s
+dapr-sentry-84565c747b-7bh8h             1/1     Running   0          35s
+dapr-sentry-84565c747b-fdlls             1/1     Running   0          41s
+dapr-sentry-84565c747b-ldnsf             1/1     Running   0          29s
+dapr-sidecar-injector-68f868668f-6xnbt   1/1     Running   0          41s
+dapr-sidecar-injector-68f868668f-j7jcq   1/1     Running   0          29s
+dapr-sidecar-injector-68f868668f-ltxq4   1/1     Running   0          36s
+```
+
+
+Update pods that are running Dapr to pick up the new version of the Dapr runtime.
+```sh
+kubectl rollout restart deploy/<DEPLOYMENT-NAME>
```

Comment:
What is an example of the DEPLOYMENT-NAME? When was this set from the steps above?

Reply:
This depends on customer's application.

Reply:
Reworded it.